bulletin/index.html

```diff
@@ -16,6 +16,9 @@
 	<meta property="og:type" content="website">
 	<meta property="og:description" itemprop="description"
 		    content="MACILE Academy Bulletin is a digital platform focused on highlight programming, events and observations, stats and results from the MACILE Academy.">
+	<!-- Twitter schema -->
+  <meta name="twitter:card" content="summary_large_image">
+	<meta name="twitter:domain" content="macile.org">
```

Comment:
- Double check indentations on L20
- Update twitter domain to `https://news.macile.org`

Reply:
![twcard](https://user-images.githubusercontent.com/29107510/79605008-2a97ff00-80bd-11ea-9f0c-22dc2df08e3d.png)